experimenter/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/FormFeatureValue/validators.ts

```diff
@@ -332,3 +347,27 @@ export function schemaAutocomplete(schema: Record<string, unknown>) {
     return null;
   };
 }
+
+/**
+ * Parse from FML errors to Diagnostics
+ */
+function parseFmlErrors(view: TestableEditorView): Diagnostic[] {
+  const diagnostics: Diagnostic[] = [];
+
+  const doc = view.state.doc;
+  const text = doc.toString();
+  const from = text.indexOf("error");
+  if (from === -1) return diagnostics;
+
+  const to = from + "error".length;
+
+  diagnostics.push(createDiagnostic(from, to, "oh no!"));
+  return diagnostics;
+}
```

Comment:
For now, this is just highlighting the word "error" if it appears anywhere in the doc. This will be highlighting the specific area of text that the FML errors refer to once it's hooked up to the endpoint (#9552)